### PR TITLE
Fix invalid handling of possible null value

### PR DESCRIPTION
**Pull request by mxr576**
Developer reference can be null a developer-specific rate plan

### CODE REVIEW

File: src/Entity/Access/RatePlanAccessControlHandler.php [hunk 1]
```diff
@@ -94,11 +94,14 @@ protected function checkAccess(EntityInterface $entity, $operation, AccountInter
     // If rate plan is a developer rate plan, and the assigned developer is
     // different from account, deny access.
     if ($rate_plan instanceof DeveloperRatePlanInterface) {
-      if ($developer = $rate_plan->getDeveloper()) {
```
Comment (mxr576) [inline src/Entity/Access/RatePlanAccessControlHandler.php:97]: Variable assignment in a condition is dangerous and should be avoided.

Comment (mxr576) [reply src/Entity/Access/RatePlanAccessControlHandler.php:97]: `Error: Call to a member function getEmail() on null in Drupal\apigee_m10n\Entity\Access\RatePlanSubscriptionAccessHandler->access() (line 102 of /mnt/files/local_mount/build/web/modules/contrib/apigee_m10n/src/Entity/Access/RatePlanSubscriptionAccessHandler.php) `

**Comment by mxr576** [comment]
Hi, can somebody review this change?

File: src/Entity/Access/RatePlanAccessControlHandler.php (hunk 1 above)
Comment (mxr576) [reply src/Entity/Access/RatePlanAccessControlHandler.php:97]: This means that there was a developer, it had a developer-specific rate plan, but the developer was removed and the developer-specific rate plan remained in the system (Apigee) with a `null` developer reference.

**Comment by shishir-intelli** [comment]
Hi @mxr576,
Could you please resolve the conflicting files.
Same for #292

Thanks

**Comment by mxr576** [comment]
Hi, 
I am not sure why I am going to have time for this on this more than 1-year-old PR, but "Allow edits by maintainers" is checked, so you should be able to rebase my PR in my repo and fix these.

**Comment by phdhiren** [comment]
Closed via #391

**Pull request closed without merging**